README.md

```diff
@@ -6,6 +6,8 @@ Installing all dependencies
 
     $ npm install
 
+Note: In some cases it may be required to comment or remove the contents of the .npmrc file to install the dependencies.
```

Comment:
can you change this to - if you are having trouble running `npm install` and are outside the adsk network, please try removing the `.npmrc` file.